tools/mksurfdata_esmf/src/mkharvestMod.F90

```diff
@@ -65,7 +60,7 @@ module mkharvestMod
 contains
 !=================================================================================
 
-  subroutine mkharvest(file_mesh_i, file_data_i, mesh_o, pioid_o, constant, ntime, rc)
+  subroutine mkharvest(file_mesh_i, file_data_i, mesh_o, pioid_o, ntime, routehandle_r8, rc)
```

Comment:
@mvertens from what I can tell, `mkharvest` was already doing a version of what you suggested to me:
I believe it was declaring routhandle_r8 at the module level to make it available in subsequent calls without recreating every time. I changed it to an in/out argument per your suggestion, but I wonder now which is preferable. Let me know what you think.